schemas/dependencies/endpoint-monitoring-provider-1.yml

```diff
@@ -0,0 +1,79 @@
+---
+"$schema": /metaschema-1.json
+version: '1.0'
+type: object
+
+additionalProperties: false
+
+properties:
+  "$schema":
+    type: string
+    enum:
+    - /dependencies/endpoint-monitoring-provider-1.yml
+  labels:
+    "$ref": "/common-1.json#/definitions/labels"
+  name:
+    type: string
+  description:
+    type: string
+  provider:
+    type: string
+    enum:
+    - blackbox-exporter
+    description: The provider defines which monitoring implementation will be used for endpoints
+  timeout:
+    type: string
+    description: Timeout for the monitoring check, e.g. 10s
+  checkInterval:
+    type: string
+    description: Interval between endpoint checks, e.g. 10s
+  metricLabels:
+    "$ref": "/common-1.json#/definitions/labels"
+    description: Labels to be added to the resulting Prometheus metrics
```

Comment:
Prometheus label names cannot have any character, as `string` provides. From https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels they must match `[a-zA-Z_][a-zA-Z0-9_]*`

Reply:
fixed in c209757